dist/index.js

```diff
@@ -13637,10 +13640,17 @@ function run() {
             ];
             if (API)
                 options.push(`--api=${API}`);
-            if (API_KEY && API_SECRET)
-                options.push(`--api-key=${API_KEY}`, `--api-secret=${API_SECRET}`);
-            if (GCP_CREDENTIALS_SERVICE_ACCOUNT)
-                options.push(`--gcp-service-account=${GCP_CREDENTIALS_SERVICE_ACCOUNT}`);
+            if (ENABLE_GITHUB_ACTION_TOKEN) {
+                options.push(`--enable-github-action-token=true`);
+            }
+            else {
+                if (API_KEY && API_SECRET) {
+                    options.push(`--api-key=${API_KEY}`, `--api-secret=${API_SECRET}`);
+                }
+                if (GCP_CREDENTIALS_SERVICE_ACCOUNT) {
```

Comment:
I think we need a switch type statement here. Note, that if API_KEY and SECRET are there, then we should not push the GCP CREDENTIALS (unless if this checked somewhere else and the two cannot be set at the same time)

Reply:
This was not checked anywhere else. Fixed now.